lib/indexer/amender.rb

```diff
@@ -6,15 +6,16 @@ def initialize(index)
       @index = index
     end
 
-    def amend(link, updates)
+    def amend(document_id, updates)
       if updates.include?("link")
         raise ArgumentError, "Cannot change document the `link` attribute of a document."
```

Comment:
I found this error message confusing
Does it means `"Cannot change `link` attribute of a document."` ?